### PR TITLE
Fixes-issue-143-Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Shruti9520**
Fixes- [#143](https://github.com/mozilla/Themer/issues/143#issuecomment-371961479)

File added in this PR-
1. `CONTRIBUTING.md`

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+[issue]: https://github.com/mozilla/Themer/issues/
+[good first]: https://github.com/mozilla/Themer/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
+
+### Contributing code
+
+We use the same process for community members and paid staff.
+
+#### Fork and Clone
+
+Fork the [repository](repo) and create a clone to your personal system.
```
Comment (pdehaan) [inline CONTRIBUTING.md]: SHould this be `[repository][repo]`, since it's not a URL but a reference to L37 below?

**Comment by Shruti9520** [comment]
Hi @pdehaan,
I made the required change. Thanks for reviewing. :-)

**Comment by pdehaan** [comment]
Awesome, thanks @Shruti9520!

**Comment by Shruti9520** [comment]
@pdehaan Thank you! :-)

**Pull request merged**